PocketKit/Sources/PocketKit/Article/ArticleComponentPresenter.swift

```diff
@@ -13,14 +14,16 @@ private extension Style {
     static let imageCaption: Self = .body.sansSerif.with(size: .p3)
     
     static let codeBlock: Self = .body.monospace
+    static let bulletedPrefix: Self = .body.monospace
+    static let standardPrefix: Self = .body.sansSerif
```

Comment:
Looks like we can get rid of these declarations since they're not being used anymore